lambda-runtime/src/AWS/Lambda/Runtime.hs

```diff
@@ -13,27 +13,27 @@ run
   :: forall m . (MonadCatch m, MonadIO m)
   => (JSON.Value -> m JSON.Value)
   -> m ()
-run = forever . processEvent
+run function = do
+  config <- either throwM pure =<< liftIO (runExceptT Client.getHttpConfig)
```

Comment:
This is a really good fix. Nice. I didn't see it.

Now, that we are re-using the same HTTP manager for potentially more requests. I think we may need to fine-tune the manager to be a bit. Potentially change `managerConnCount  & managerIdleConnectionCount` values.

Reply:
there can only ever be one event be in flight. So the current values are exactly right.